### PR TITLE
SNe Id, Idn, Ie, Ien, and LRN

### PR DESCRIPTION
**Pull request by Theodlz**
This PR adds:
-  a subtype of Novae: Luminous Red Nova (LRN)
- 4 new SNe subtypes: Id, Idn, Ie, Ien

This is based on user requests, that need this in the Sitewide taxonomy of SkyPortal (that reads its content from this repo's main branch).

### CODE REVIEW

**Comment by Theodlz** [comment]
Feel free to make any changes!

**Comment by Theodlz** [comment]
@profjsb

File: tdtax/supernovae.yaml [hunk 1]
```diff
@@ -134,3 +134,25 @@ subclasses:
   tags: [intermediate luminosity red transient]
   links:
     - https://arxiv.org/abs/1908.02323
+- class: Id
+  tags: [hydrogen poor]
+  comments: |
+    SNe with O/Ne/Mg-rich (and C/He/H-poor) outer layers
+  other names: [SN Id, SNId]
+  links:
+    - https://arxiv.org/pdf/2409.02054
```
Comment (profjsb) [inline tdtax/supernovae.yaml:143]:
```suggestion
    - https://arxiv.org/abs/2409.02054
```

File: tdtax/supernovae.yaml [hunk 2]
```diff
@@ -134,3 +134,25 @@ subclasses:
...
+    - class: Idn
+      tags: [interacting, narrow line]
+      other names: [SN Idn, SNIdn]
+- class: Ie
+  tags: [hydrogen poor]
+  comments: |
+    SNe with O/Si/S-rich (and Mg/Ne/He/H-poor) outer layers
+  other names: [SN Ie, SNIe]
+  links:
+    - https://arxiv.org/pdf/2409.02054
```
Comment (profjsb) [inline tdtax/supernovae.yaml:154]:
```suggestion
    - https://arxiv.org/abs/2409.02054
```

**Review by profjsb — CHANGES_REQUESTED** [review]
Looks good. Tiny change requested.

**Review by profjsb — APPROVED** [review]
LGTM

**Pull request merged**